src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -460,12 +442,13 @@ class BaseCourseCard extends Component {
                 </div>
                 {this.renderSettingsDropdown(dropdownMenuItems)}
               </div>
+              {this.renderCourseInfoOutline()}
               {this.renderButtons()}
               {this.renderChildren()}
               <div className="course-misc-text row">
                 <div className={`col ${isExecutiveEducation2UCourse ? 'text-light-300' : 'text-gray'}`}>
```

Comment:
nit/unrelated: could replace these div's with `Row` and `Col` components, removing the `row` and `col` class names.